### ISSUE TITLE
Solution: working-missing libexpat.dll on windows 7 x64

### ISSUE THREAD

**Issue by loomisraider** [koying/SPMC#673]
installed SPMC on a windows 7 x64.  Windows said the program was missing a libexpat.dll, try reinstalling the program.  

Solution:  copy the original libexpat.dll file from C:\Program Files (x86)\SPMC\system to the SPMC folder which is C:\Program Files (x86)\SPMC

No need to delete the original libexpat.dll file from original folder.

**Comment by koying** [comment]
Strange, I had fixed that one.
Will look again.

**Comment by unlocKing** [comment]
@loomisraider Thanks, I've had the same issue.

**Comment by ghost** [comment]
yes this was the case for me too.. windows 10... thanks @loomisraider
